stage1/tasks/hangman/README.md

```diff
@@ -0,0 +1,88 @@
+# Hangman
+
+## Task
+### Project's Description
+
+[Hangman](https://en.wikipedia.org/wiki/Hangman_(game)) is a classic word game in which you must find the correct answer by guessing guessing letters one at a time.
+
+### Game rules
+1. The game is represented by two main parts: the gallows part and the quiz part.
+2. At the beginning of the game, the gallows is empty.
+3. The quiz part includes:
+- a question,
+- a secret word (the answer to the question) represented by the blank lines for each letter in the word *(for example, if the secret word is 'address' it will be written like '_ _ _ _ _ _ _')*,
+- an incorrect guesses counter,
+- a virtual keyboard,
+4. The user should be able to use **both virtual** *(by clicking letter buttons)* and **physical keyboards** *(by pressing letter keys)* for playing the game.
+5. When the user starts guessing the letters:
+- whenever the letter is guessed correctly, this letter appears instead of the corresponding blank line *(for the example mentioned above, if the user clicks or press on the 'd' letter, both 'd's should appear on the screen: '_ d d _ _ _ _')*;
+- if the letter is not in the word, **one body part** should be added to the gallows and the incorrect guesses counter should be updated accordingly;
+- the clicked/pressed letter should be disabled and it should be reflected in the UI *(for example, by using a different color on the disabled button)*;
+6. The body parts appear on the gallows in the following order: head, body, left arm, right arm, left leg, right leg. It means that the user has *6 attempts* to guess the word.
+7. The player will continue guessing letters until either the word is solved or all six body parts are on the gallows.
+8. At the end of the game, a modal window should appear. It should:
+- include the message that either congratulates or informs the user that the game has been lost;
+- the secret word (independently whether it was guessed correctly or not);
```

Comment:
_It's better to paraphrase the word "independently" as long as it doesn't convey the necessary idea completely_

**Corrected (simpler option 1):** - the secret word (even if it was guessed correctly);
**Corrected (option 2):** - the secret word (regardless of the fact whether or not it was guessed correctly);

Reply:
Done